be/src/exec/olap_utils.h

```diff
@@ -207,6 +207,9 @@ inline SQLFilterOp to_olap_filter_type(TExprOpcode::type type, bool opposite) {
     case TExprOpcode::NE:
         return opposite ? FILTER_IN : FILTER_NOT_IN;
 
+    case TExprOpcode::EQ_FOR_NULL:
```

Comment:
Can the storage engine handle `EQ_FOR_NULL` correctly with `FILTER_IN`?

Reply:
It's OK. 
case 1:
```sql
select * from t1 where k1 <=> 1;
```
It's equals to EQ

case 2:
```
select * from t1 where k1 <=> null;
```
type  of k1 will not equals to LITERAL_NULL,  so storage engine will not use this conjunct